README.md

```diff
@@ -187,6 +187,45 @@ Finally,  re-run the project playbook, limiting the scope to the machine that yo
 $ OULIB_USER=vagrant vagrant ssh ansible --command "ansible-playbook /vagrant/projects/web/playbooks/vagrant.yml --limit solr.vagrant.localdomain"
 ```
 
+Ad-hoc Ansible Commands
+------
+As this is an attempt to replicate the production enviornment as closely as possible ad-hoc Ansible commands are not immediately available and require one to SSH into a specific vagrant box within the prod-infrastructure. Below are two scenarios:
+
+### Scenario #1:
+* User follows all preceding steps
+* User Git clones this archive
+* User enters vagrant_infrastructure directory on their box
+* User initializes boxes and enviornment
+```
+	user@localdomain$: vagrant up
+```
+* User attempts to run an adhoc command to gather information about web server/host
+```
+	user@localdoamin$: ansible web -m setup
+```
+This scenario will result in an error if one were to Git clone this archive as the host itself is not defined on the machine running ansible.
+
+### Scenario #2:
+* User follows all preceding steps
```

Comment:
I was slightly confused about by what "preceding steps" referred to.  I might change to "ensure all requirements are met" or something along those lines.